ProjFS.Mac/Scripts/Build.sh

```diff
@@ -21,9 +21,71 @@ fi
 
 # Run Tests and put output into a xml file
 set -o pipefail
-xcodebuild -configuration $CONFIGURATION -project $PROJFS/PrjFS.xcodeproj -scheme 'Build All' test | xcpretty -r junit --output $PROJFS/TestResultJunit.xml || exit 1
+xcodebuild -configuration $CONFIGURATION -enableCodeCoverage YES -project $PROJFS/PrjFS.xcodeproj -derivedDataPath $ROOTDIR/BuildOutput/ProjFS.Mac/Coverage -scheme 'Build All' test | xcpretty -r junit --output $PROJFS/TestResultJunit.xml || exit 1
```

Comment:
nit: Do we not want to extract a local variable for the `derivedDataPath` argument value?

Reply:
Good point yes!